charts/prometheus-stackdriver-exporter/templates/deployment.yaml

```diff
@@ -4,6 +4,10 @@ metadata:
   name: {{ template "stackdriver-exporter.fullname" . }}
   labels:
     {{- include "stackdriver-exporter.labels" . | indent 4 }}
+    {{- with .Values.annotations }}
```

Comment:
I'm not sure, but looks like you should use "{{- if .Values.annotations  }}" to exclude error

```
  Error: template: prometheus-stackdriver-exporter/templates/deployment.yaml:9:24: executing "prometheus-stackdriver-exporter/templates/deployment.yaml" at <.Values.annotations>: nil pointer evaluating interface {}.annotations
  ```